tests/e2e_tests/test_data_compatibility.py

```
@@ -0,0 +1,150 @@
+        missing = expected - found
+        if missing:
+            print(f"Missing expected collections: {sorted(missing)}")
+            return False
+
+        for collection in expected:
+            try:
+                collection_info = client.get_collection_info_dict(collection)
+                if collection_info["status"] != "ok":
+                    print(f"Collection {collection} returned status {collection_info['status']}")
+                    return False
+            except Exception as error:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)